src/docs/Components/Form/Datepicker/code.mdx

```diff
@@ -5,31 +5,47 @@ order: 2
 
 ## Import
 
-Import the **settings** and the **text-input** `scss` files.
+Import the **settings**, the **text-input** and the **field** `scss` files.
 
 ```scss
 @import 'settings-tools/_all-settings';
 
 @include import-font-families();
 
 @import 'components/_c.text-input';
+
+@import 'components/_c.fields';
 ```
 
-## Basic usage
+## Usage
 
-The easiest way to implement a date picker on a form field is to use the `type="date"` attribute on your `input` element.
+The easiest way to implement a date picker on a form field is to apply the following elements:
 
-In addition, the `input` element must also have the `mc-text-input` class.
+- add the `mc-field` class on a html **div** element
+- add the `mc-field__label` class on a **label** element
+- add the `mc-field__input`class in addition of the `mc-text-input` class on an **input** element
+- use the `type="date"` attribute on your **input** element
 
 ```html
-<input type="date" class="mc-text-input" placeholder="Empty" name="example" />
+<div class="mc-field">
+  <label class="mc-field__label" for="example">
+    Label
+  </label>
+
+  <input
+    type="date"
+    class="mc-text-input mc-field__input"
+    id="example"
+    name="example"
+  />
+</div>
 ```
 
 <Highlight theme="warning" title="date picker appearance">
 
-  Using the `type="date"` attribute on an `input` element triggers the automatic generation of a date picker by the browser. The appearance of this date picker will therefore depend on the browser used.
+  Using the `type="date"` attribute on an `input` element triggers the automatic generation of a date picker by the browser. The appearance and overall functionality of this date picker will therefore depend on the browser used.
 
-  To solve this problem, you will have to implement a JavaScript library of your choice (see below "Custom date picker").
+  So, if you want to have advanced control over appearance and functionality, you will have to use a JavaScript library for that (see below "[Custom usage](/Components/Form/Datepicker/code/#custom-usage)").
```

Comment:
=> `To have advanced control over appearance and features...`

Reply:
fixed